substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -410,7 +420,13 @@ impl<B: ChainApi, L: EventHandler<B>> Pool<B, L> {
 			self.verify(at, pruned_transactions, CheckBannedBeforeVerify::Yes).await;
 
 		let pruned_hashes = reverified_transactions.keys().map(Clone::clone).collect::<Vec<_>>();
-		log::debug!(target: LOG_TARGET, "Pruning at {:?}. Resubmitting transactions: {}, reverification took: {:?}", &at, reverified_transactions.len(), now.elapsed());
+		debug!(
+			target: LOG_TARGET,
+			?at,
+			reverified_transactions = reverified_transactions.len(),
+			elapsed = ?now.elapsed(),
+			"Pruning at. Resubmitting transactions."
```

Comment:
```suggestion
			"prune_tags: reverification done and now resubmitting same transactions"
```

Reply:
let’s keep it as is.

Reply:
The thing that throws me off is the `elapsed` field and then seeing the log for `resubmitting transactions`, where one doesn't have to do with the other. Should we change at least  `elapsed` to `reverification_duration`?

Reply:
It shall be simply duration. I added other suggestion.

Reply:
OK, we can changed it to, would it be fine for you?
```suggestion
			"Pruned. Resubmitting transactions."
```

Reply:
Ah, I see - in practice elapsed corresponds to the `prune` logic. Your suggestion is better.